delta/testkit/src/main/scala/ch/epfl/bluebrain/nexus/testkit/TestHelpers.scala

```diff
@@ -28,6 +28,15 @@ trait TestHelpers extends ClasspathResourceUtils {
     inner("", length)
   }
 
+  /**
+    * Generate the cross product of the two given sequences
+    */
+  def crossProduct[A, B](first: Seq[A], second: Seq[B]): Seq[(A, B)] =
```

Comment:
isn't the following equivalent?
https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/Apply.scala#L112

Reply:
It somehow clashed with a product method in scala.collection.
But (first, second).tupled do it fine